examples/experiments/progress_bar.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+================
+ProgressBar demo
+================
+
+This example shows how to display progress between trials using
+:class:`expyfun.visual.ProgressBar`.
+"""
+from expyfun import ExperimentController, building_doc
+from expyfun.visual import ProgressBar
+import expyfun.analyze as ea
+import numpy as np
+
+n_trials = 6
+max_wait = 0.1 if building_doc else np.inf
+wait_dur = 0.1 if building_doc else 0.5
+
+with ExperimentController('name', version='dev', window_size=[800, 600],
+                          full_screen=False, session='foo',
+                          participant='foo') as ec:
+
+    # initialize the progress bar
+    pb = ProgressBar(ec, [0, -.1, 1.5, .1], units='norm')
+
+    ec.screen_prompt('Press the number shown on the screen. Start by pressing'
+                     ' 1.', font_size=16, live_keys=[1], max_wait=max_wait)
+
+    for n in np.arange(n_trials) + 1:
+        # subject does some task
+        number = np.random.randint(1, 5)
+        ec.screen_text(str(number), wrap=False)
+        ec.flip()
+        ec.wait_one_press(live_keys=[number], max_wait=max_wait)
+        ec.flip()
+        ec.wait_secs(wait_dur)
+        # only show progress bar every other trial
+        if n % 2 == 0:
+            # calculate percent done and update the bar object
+            percent = int(n * 100 / n_trials)
```

Comment:
why `int`? `update_bar` should be able to handle `float`

Reply:
Oh that's just because I was using it in the text too and didn't want it to say "You completed 33.33333333333333333 %". But there are better ways to do that. I'll fix it first thing tomorrow morning.

Reply:
Yeah it would be better to do it just in the text display, but this is fine for now (I was worried there was some limitation I missed)